main.cc

```diff
@@ -827,6 +827,7 @@ run()
 
         SDL_Event e;
         while (SDL_PollEvent(&e)) {
+            ImGui::SetCurrentContext(default_context);
```

Comment:
Solidly the right thing :)

Reply:
Actually -- perhaps this is better done a couple lines up? Nothing in this loop should be disturbing the context.

Reply:
Sure, done.